.github/workflows/niployments.yml

```diff
@@ -0,0 +1,23 @@
+name: Deploy
+
+on:
+  push:
+    branches:
+      - main
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Upload to NIployments register
+      uses: NIAEFEUP/push-to-niployments@main
```

Comment:
We should be using release tags here, not branch names. Just a minor nitpick so we don't get hurt later on when the action is further developed.

Reply:
I just copied the suggested usage from the action repository... Maybe it should be updated there as well!

Reply:
Using branches is not recommended, I would say that we shouldn't use tags either, to avoid supply chain attacks, but because it's internal, it's fine.

Reply:
Tags are the recommended way by GitHub, are they not?

Reply:
They are the convenient way eg.: you get fixes but are inherently vulnerable to supply chain attacks. If you need hardening you should use [fixed SHAs](https://docs.github.com/en/actions/security-guides/security-hardening-for-github-actions#using-third-party-actions). Just an observation that is not relevant to our use-case :sweat_smile:

Reply:
I've gone ahead and used the v2.1 tag (yet to be released)